src/main/java/uk/gov/hmcts/dts/fact/entity/Court.java

```diff
@@ -67,7 +58,7 @@
     )
     private List<CourtType> courtTypes;
 
-    @OneToMany(cascade = CascadeType.ALL, mappedBy = COURT_STRING, orphanRemoval = true)
+    @OneToMany(mappedBy = COURT_STRING, orphanRemoval = true)
```

Comment:
Needed to be removed, as conflicts with save/delete operations for new emails. The persist within the cascade all reverts all changes made automatically